sdk/eventhub/Azure.Messaging.EventHubs/src/Processor/PartitionPump.cs

```diff
@@ -225,64 +225,67 @@ private async Task RunAsync(CancellationToken cancellationToken)
 
             while (!cancellationToken.IsCancellationRequested)
             {
-                try
+                await using (var partitionReceiver = InnerConsumer.CreatePartitionReceiver(Context.PartitionId, StartingPosition))
```

Comment:
This is temporary and likely is the cause of the issues with the Live tests that I ignored;  since its being rewritten, I did not sped time debugging more closely.